### PR TITLE
Added new Concepts, Explore and Manage topics

### PR DESCRIPTION
**Pull request by royendo**
Need some feedback on the following changes, Okay to proceed with creating some content
1) Reorganizing the sidebar,
    Concepts:
     - Rill Cloud vs Rill Developer
     - What is an OLAP engine (contents from Connect OLAP engine)

2) Explore
         - Sharing with public URL
                  - some notes inline, will add content (gifs) but thoughts on article as a whole?

3) Added a reference to our tutorials page and a brief summary, can be hidden until Tutorial is ready.

4) Added mock-users to rill-yaml.md from <manage/security#testing-your-policies> w/screenshot

### CODE REVIEW

**Comment by royendo** [comment]
@AndrewRTsao ,
when you get the chance, can you review the following for additional docs?
Added articles:
    Concepts:
    - Rill Cloud v Rill Developer
    - What is an OLAP Engine? [moved from build section]
    
    Explore:
    - Sharing with public URL
    
    Manage:
    - User Group Management,
    - Project Management. (thoughts on graphic?)

**Comment by AndrewRTsao** [comment]
Will definitely try to take a look later this week. Thanks Roy!

**Comment by royendo** [comment]
Added/modified  articles:
Concepts:
- Rill Cloud v Rill Developer
- What is an OLAP Engine? [moved from build section]
Deploy:
- Added UI deploy
- moved some content to project / org managment
Explore:
- Sharing with public URL
Manage:
- User Group Management,
- Org/ Project Management. (thoughts on graphic?)

File: docs/docs/home/.tutorial.md
Comment (AndrewRTsao) [inline docs/docs/home/.tutorial.md:1]: @royendo I'm guessing this is still a WIP and meant to be a skeleton / not ready to ship yet?

**Review by AndrewRTsao — APPROVED** [review]
Made some edits, cleaned up some sections / typos, and rewrote a few parts. Good to push - thanks for the work Roy!

**Comment by AndrewRTsao** [comment]
Unless you also want to include tutorials as part of this, I'm comfortable pushing these changes @royendo

**Comment by AndrewRTsao** [comment]
@royendo Will merge these changes for now and we can tackle / segment out tutorials separately

**Pull request merged**